sharktank_models/test_suite/regression_tests/test_model_threshold.py

```diff
@@ -0,0 +1,196 @@
+# Copyright 2025 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import pytest
+from ireers_tools import *
+import os
+from conftest import VmfbManager
+from pathlib import Path
+import subprocess
+import json 
+
+rocm_chip = os.getenv("ROCM_CHIP", default="gfx942")
+vmfb_dir = os.getenv("TEST_OUTPUT_ARTIFACTS", default=Path.cwd()) 
+sku = os.getenv("SKU", default="mi300")
+
+# Helper methods
+def fetch_source_fixtures_for_run_flags(inference_list, model_name, submodel_name):
+    result = []
+    for entry in inference_list:
+        source = entry.get("source")
+        value = entry.get("value")
+        source_fixture = fetch_source_fixture(source, group=f"{model_name}_{submodel_name}")
+        result.append([source_fixture.path, value])
+    
+    return result
+    
+def common_run_flags_generation(input_list, output_list):
+    flags_list = []
+
+    if input_list:
+        for path, value in input_list:
+            if not value:
+                flags_list.append(f"--input=@{path}")
+            else:
+                flags_list.append(f"--input={value}=@{path}")
+
+    if output_list:
+        for path, value in output_list:
+            if not value:
+                flags_list.append(f"--expected_output=@{path}")
+            else:
+                flags_list.append(f"--expected_output={value}=@{path}")
+    
+    return flags_list
+
+
+class TestModelThreshold:
+    @pytest.fixture(autouse = True)
+    @classmethod
+    def setup_class(self, pytestconfig):
+        self.model_name = pytestconfig.getoption("model_name")
+        self.submodel_name = pytestconfig.getoption("submodel_name")
+
+        file_name = f"{Path.cwd()}/sharktank_models/test_suite/regression_tests/{self.model_name}/{self.submodel_name}.json"
```

Comment:
another brittle use of `Path.cwd()`, recommend using the path to the current script file and then walking up/down from there